datasets/th/designated_person/th_designated_person.yaml

```
@@ -0,0 +1,23 @@
+title: Thailand's Designated Persons List
+entry_point: crawler.py
+prefix: th-des
+coverage:
+  frequency: daily
+  start: 2024-04-08
+load_db_uri: ${OPENSANCTIONS_DATABASE_URI}
+summary: >
+  List of Designated Persons from Thailand's AMLO
+description: |
+  This dataset comprises information about designated persons published by
+  Thailand's Anti Money Laundring Office (AMLO)
+  under the Counter-Terrorism and Proliferation
+  Financing Act B.E. 2559 section 7
```

Please wrap at 80 chars

```suggestion
  Financing Act B.E. 2559 section 7
  
  According to the act:
  
    > ...[R]eporting entities and persons in possession of the assets of the designated persons [will be informed to]  Freeze the assets of the designated persons or a person acting on behalf of, or at the direction of, or an undertaking owned or controlled by such persons, directly or indirectly
```